portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase

```diff
@@ -949,4 +949,116 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-129536. View the configuration of widgets on content page persist after execute Clean Up Orphaned Theme Portlet Preferences."
+	@priority = "4"
+	test ViewWidgetsConfigurationOnContentPagePersistAfterCleanUpOrphanedTheme {
+		property test.name.skip.portal.instance = "ContentPages#ViewWidgetsConfigurationOnContentPagePersistAfterCleanUpOrphanedTheme";
+
+		task ("Add a navigation menu") {
+			JSONSitenavigation.addSiteNavigationMenu(
+				groupName = "Test Site Name",
+				siteNavigationMenuName = "Navigation Menu Name");
+		}
+
+		task ("Navigate to the navigation menu") {
+			NavigationMenusAdmin.openNavigationMenusAdmin(siteURLKey = "test-site-name");
+
+			LexiconEntry.gotoEntry(rowEntry = "Navigation Menu Name");
+		}
+
+		task ("Add an URL type item to navigation menu") {
+			NavigationMenusAdmin.addItem(
+				item = "URL",
+				url = "https://www.liferay.com/",
+				urlName = "Liferay");
+		}
+
+		task ("Add a web content via JSON") {
+			JSONWebcontent.addWebContent(
+				content = "Web Content Content",
+				groupName = "Test Site Name",
+				title = "Web Content Title");
+		}
+
+		task ("Add a Asset Publisher widget to the page") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Content Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.addWidget(portletName = "Asset Publisher");
+		}
+
+		task ("Select the web content as manual selection in Asset Publisher") {
+			AssetPublisherPortlet.configureManualAssetSelectionPG(
+				assetTitle = "Web Content Title",
+				assetType = "Basic Web Content");
+		}
+
+		task ("View web content is selected in Asset Publisher") {
+			SelectFrame.selectFrameNoLoading(
+				key_title = "Asset Publisher",
+				locator1 = "IFrame#MODAL_ANY");
+
+			AssertTextEquals(
+				key_assetTitle = "Web Content Title",
+				locator1 = "AssetLists#ASSET_TITLE_ROW",
+				value1 = "Web Content Title");
+
+			IFrame.closeFrame();
+		}
+
+		task ("Add a Navigation Menu widget to the page") {
+			PageEditor.addWidget(
+				portletName = "Navigation Menu",
+				targetFragmentName = "Asset Publisher");
+		}
+
+		task ("Configure the Navigation Menu widget to use the navigation menu") {
+			NavigationMenusWidget.gotoNavigationMenuWidgetConfig();
+
+			NavigationMenusWidget.chooseMenu(menuName = "Navigation Menu Name");
+
+			Portlet.saveAndCloseConfig();
+		}
+
+		task ("View the Navigation Menu at edit mode") {
+			AssertVisible(
+				key_pageName = "Liferay",
+				locator1 = "Home#PAGE");
+
+			PageEditor.clickPublish();
+		}
+
+		task ("Execute Clean Up Orphaned Theme Portlet Preferences in Server Administration") {
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoPortlet(
+				category = "System",
+				panel = "Control Panel",
+				portlet = "Server Administration");
+
+			ServerAdministration.executeServerResourcesActions(actionsDescription = "Clean Up Orphaned Theme Portlet Preferences");
+		}
+
```

Comment:
Add steps to view web content displayed at Display Template in Asset Publisher at view mode